projects/observability/src/shared/components/iframe/iframe.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+
+@Component({
+  selector: 'ht-iframe',
+  template: `<iframe width="100%" height="100%" frameBorder="0" [src]="urlSafe"></iframe>`,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./iframe.component.scss']
+})
+export class IFrameComponent implements OnInit {
+  @Input() public source: string = '';
+  @Input() public title?: string;
+
+  public urlSafe?: SafeResourceUrl;
+  public constructor(public sanitizer: DomSanitizer) {}
+
+  public ngOnInit(): void {
```

Comment:
any time source changes, we'd want to rerun this - it should be on changes rather than on init.